src/content/plugins/chunks-webpack-plugin.md

```diff
@@ -0,0 +1,322 @@
+
+```js
+new ChunksWebpackPlugin({
+  templateScript: '<script src="{{chunk}}"></script>'
+});
+```
+
+T> Keep the `{{chunk}}` placeholder, it is automatically replaced by the concatenation of the webpack public path and the chunk filename.
+
+### `customFormatTags`
+
+`false || function (chunksSorted, files)`
+
+Tells the plugin whether to personalize the default behavior for generate your own templates. The function is called for each entry points. Custom behavior can also be add for a specific entry point if necessary.
+
+```js
+new ChunksWebpackPlugin({
+  customFormatTags: (chunksSorted, Entrypoint) => {
+    // Generate all HTML style tags with a CDN prefix
+    const styles = chunksSorted.styles.map(chunkCss =>
+      `<link rel="stylesheet" href="https://cdn.domain.com/${chunkCss}" />`
+    ).join('');
+
+    // Generate all HTML style tags with CDN prefix and defer attribute
+    const scripts = chunksSorted.scripts.map(chunkJs =>
+      `<script defer src="https://cdn.domain.com/${chunkJs}"></script>`
+    ).join('');
+
+    return { styles, scripts };
+  }
+});
+```
+
+T> The arrow function syntax allow you to access the class properties.
+
+W> The function provides more flexibility by replacing the default behavior. Follow the example above to make sure it works.
+
+The function must return an object with the following format:
+
+```json
+{
+  "styles": "",
+  "scripts": ""
+}
+```
+
+The `customFormatTags` function has two parameters:
+
+#### `chunksSorted`
+
+`object`
+
+The list of the chunks sorted by type: `styles` and `scripts`.
+
+#### `Entrypoint`
+
+`Object`
+
+The object is included in every single ChunkGroup. The variable contains all information about the current entry point; log it on the console for more details.
+
+W> Use this variable only for a full customization if the `chunksSorted` variable does not meet your needs.
+
+### `generateChunksManifest`
+
+`boolean = false`
+
+Tells the plugin whether to generate the `chunks-manifest.json`. The file contains the list of all the chunks grouped by entry points.
+
+```js
+new ChunksWebpackPlugin({
+  generateChunksManifest: false
+});
+```
+
+Example of the output of the `chunks-manifest.json` file:
+
+```json
+{
+  "app-a": {
+    "styles": ["/dist/css/vendors~app-a~app-b.css", "/dist/css/app-a.css"],
+    "scripts": ["/dist/js/vendors~app-a~app-b.js", "/dist/js/app-a.js"]
+  },
+  "app-b": {
+    "styles": ["/dist/css/vendors~app-a~app-b.css", "/dist/css/app-b.css"],
+    "scripts": ["/dist/js/vendors~app-a~app-b.js", "/dist/js/app-b.js"]
+  }
+}
+```
+
+### `generateChunksFiles`
+
+`boolean = true`
+
+Tells the plugin whether to generate the HTML files.
+
+```js
+new ChunksWebpackPlugin({
+  generateChunksManifest: true
+});
+```
+
+W> This option cancels the main functionality of the plugin and HTML files will not be generated. It can be useful only with addition of `generateChunksManifest` option for a custom generation of the HTML files.
```

Comment:
```suggestion
W> When set to `false`, HTML files will not be generated. It can only be useful together with  `generateChunksManifest` option set to `true` for custom generation of the HTML files.
```